packages/lru_cache/lru_cache.0.4.0/opam

```diff
@@ -32,9 +32,9 @@ build: [
 dev-repo: "git+https://framagit.org/zoggy/ocaml-lru-cache.git"
 url {
   src:
-    "https://framagit.org/zoggy/ocaml-lru-cache/-/archive/0.4.0/ocaml-lru-cache-0.4.0.tar.bz2"
+    ""https://zoggy.frama.io/ocaml-lru-cache/releases/ocaml-lru-cache-0.4.0.tar.gz
```

Comment:
Can we stick to the tar bz2 and avoid changing the checksum for this one?

Reply:
I have manually compared the two tarballs, so I know that the content is the same. It is mostly to avoid breaking other people code (for example, often changing checksums breaks nix users, not sure if this is the case here since you also change file name)